### PR TITLE
Add public group getter for java and scala

### PR DESCRIPTION
**Pull request by stpyang**

### CODE REVIEW

**Comment by hcho3** [comment]
@stpyang Thanks for your contribution. Take a look at our contribution guideline: https://xgboost.readthedocs.io/en/latest/contrib/index.html.

**Comment by hcho3** [comment]
@stpyang I see that you've force-pushed the pull request 4 times in a now without any change. Is there something your're trying to accomplish?

**Comment by stpyang** [comment]
No, sorry, I was just catching typos that i wanted to fix.

**Comment by hcho3** [comment]
@stpyang Is this ready now?

**Comment by stpyang** [comment]
Yes

File: jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/DMatrix.java [hunk 1]
```diff
@@ -204,6 +204,15 @@ public void setGroup(int[] group) throws XGBoostError {
     XGBoostJNI.checkCall(XGBoostJNI.XGDMatrixSetGroup(handle, group));
   }
 
+  /**
+   * Get group sizes of DMatrix
+   *
+   * @throws XGBoostError native error
+   */
+  public int[] getGroup() throws XGBoostError {
+    getIntInfo("group_ptr")
```
Comment (hcho3) [inline jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/DMatrix.java]:
```suggestion
    getIntInfo("group_ptr");
```

**Comment by hcho3** [comment]
@stpyang How did you test your proposed changes?

File: jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/DMatrix.java (hunk 1 above)
Comment (stpyang) [reply jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/DMatrix.java]: Done.

**Comment by stpyang** [comment]
This is a subset of some forked code that we use in production at my company.

**Comment by stpyang** [comment]
I altered the name of this commit from `Add public group getter for java` to `Add public group getter for java and scala`

**Comment by hcho3** [comment]
@stpyang We're merging this now. Thanks for your contribution.

**Pull request merged**